eng/pipelines/common/templates/runtimes/run-test-job.yml

```diff
@@ -237,6 +237,8 @@ jobs:
       timeoutInMinutes: 510
     ${{ if eq(parameters.testGroup, 'jitstress-isas-x86') }}:
       timeoutInMinutes: 960
+    ${{ if and( eq(parameters.runtimeFlavor, 'mono'), eq(parameters.runtimeVariant, 'llvmfullaot') }}:
+      timeoutInMinutes: 400
```

Comment:
looks like this doesn't work: https://dev.azure.com/dnceng/public/_build/results?buildId=1918800&view=results

I think this is because we're already going into the if case above that sets `timeoutInMinutes: 200`, you'll need to add a case there. Or alternatively remove the `eq(parameters.osGroup, 'windows')` so we always set 300min on arm64

Reply:
I changed it to just remove the `eq(parameters.osGroup, 'windows')` for arm64